requirements.in

```diff
@@ -21,5 +21,7 @@ seaborn
 
 spacy # nlp
 
+# for downloading csvs
+datetime
```

Comment:
datetime is standard to Python! No need to include it (unless there's a feature you need in a version of datetime that I'm not thinking of?)